spec/presenters/change_history_presenter_spec.rb

```diff
@@ -1,83 +1,79 @@
 RSpec.describe Presenters::ChangeHistoryPresenter do
-  let(:document) { create(:document) }
+  let(:document) { build(:document) }
   let(:edition) do
-    create(
+    build(
       :edition,
       document:,
       details: details.deep_stringify_keys,
     )
   end
-  let(:details) { {} }
   subject { described_class.new(edition).change_history }
 
   describe "#change_history" do
-    context "details hash includes content_history" do
-      let(:details) do
-        { change_history: [
-          { public_timestamp: 1.day.ago.to_s, note: "note 1" },
-          { public_timestamp: 2.days.ago.to_s, note: "note 2" },
-        ] }
-      end
-      it "returns content_history from details hash" do
-        expect(subject).to eq details[:change_history]
-      end
-    end
+    context "when an edition does not have a change history" do
+      let(:details) { {} }
+      let(:change_notes) { build_list(:change_note, 3) }
 
-    context "details hash doesn't include content_history" do
       before do
-        2.times do |i|
-          create(:change_note, edition:, note: i.to_s, public_timestamp: Time.zone.now.utc)
-        end
+        expect(Presenters::Queries::ChangeHistory).to receive_message_chain(:new, :call)
+                                                        .with(edition, include_edition_change_history: true)
+                                                        .with(no_args)
+                                                        .and_return(change_notes)
       end
-      it "constructs content history from change notes" do
-        expect(subject.map { |item| item[:note] }).to eq %w[0 1]
-      end
-    end
 
-    it "orders change notes by public_timestamp (ascending)" do
-      [1, 3, 2].to_a.each do |i|
-        create(:change_note, edition:, note: i.to_s, public_timestamp: i.days.ago)
+      it "returns the change notes from the database" do
+        expect(subject).to eq([
+          { public_timestamp: change_notes[0].public_timestamp.as_json, note: change_notes[0].note },
+          { public_timestamp: change_notes[1].public_timestamp.as_json, note: change_notes[1].note },
+          { public_timestamp: change_notes[2].public_timestamp.as_json, note: change_notes[2].note },
```

Comment:
In this test, the timestamp is being rendered `as_json`, whereas in the other tests it is `.utc.to_s`.  Do these return different things?